packages/fluentui/accessibility/src/behaviors/Tooltip/tooltipAsLabelBehavior.ts

```diff
@@ -10,30 +14,21 @@ import tooltipAsDescriptionBehavior, { TooltipBehaviorProps } from './tooltipAsD
  */
 const tooltipAsLabelBehavior: Accessibility<TooltipBehaviorProps> = props => {
   const behaviorData = tooltipAsDescriptionBehavior(props);
-  const defaultAriaLabeledBy = getDefaultAriaLabelledBy(props);
+  const triggerAriaLabel = props.triggerAriaLabel || props['aria-label'];
```

Comment:
I am not sure here if should we take into the "triggerAriaLabel" the "aria-label" prop which comes from `Tooltip` ? 

When I tried to simulate, then "aria-label" which was set on the `Tooltip` is set on the trigger button. (of course different question is why would someone set aria-label on the  `Tooltip`)
![image](https://user-images.githubusercontent.com/22620150/78637976-c484ce00-78ab-11ea-8eb7-b6c4cfad771d.png)

Reply:
if there is a aria-label on the tooltip, it should just stay on the tooltip and should not be applied on the trigger I think

Reply:
I am now considering aria-label from trigger when I am conditionally applying aria-labelledby.

As for that classname, I fixed it by removing the classname and using another selector in the test. should be fine now